backend/main_page/views/chatgpt_description_view.py

```diff
@@ -0,0 +1,40 @@
+from django.views import View
+from rest_framework.response import Response
+from rest_framework import status
+from rest_framework.request import Request
+from openai import OpenAIAPI
+
+API_OPENAI="https://api.openai.com/v1/chat/completions"
+
+class ChatGPTDescription(View):
+
+    def get(self, request: Request,word: str) -> Response:
+
+        word_searched = request.query_params.get("search", None)
+        description = self.get_gpt_description(word)
```

Comment:
I didn't even use word_searched,  I used the word received in the parameter named "word". I guess I can delete line 13 completely

Reply:
Yes, but in the near future the get call will not receive any parameters because every component from the front end will call their specific back end view

Reply:
ok, understood

Reply:
Change of plans, leave it as it is.

Reply:
ok,sure

Reply:
Sorry for the confusion :point_right: :point_left:

Reply:
it's fiiiiine